tests/python/test_sparse_matrix.py

```diff
@@ -0,0 +1,182 @@
+import taichi as ti
```

Comment:
You probably need to test `matmul` with operands having different shapes. E.g., a `2x3` matrix `matmul`'ed with a `3x4` matrix.

Reply:
I remember you said this before. The last test  is used to test `2x3` matrix `matmul`ed with a `3x2` matrix.

Reply:
I see, nice! The test would be stronger if the second matrix is `3x4`. Thanks!

Reply:
OK, I would update the test.